tests/LogsActivityTest.php

```diff
@@ -9,206 +9,247 @@
 
 class LogsActivityTest extends TestCase
 {
-    /** @var \Spatie\Activitylog\Test\Article|\Spatie\Activitylog\Traits\LogsActivity */
-    protected $article;
+	/** @var \Spatie\Activitylog\Test\Article|\Spatie\Activitylog\Traits\LogsActivity */
```

Comment:
This has been solved. I've added a Spatie-specific PhpStorm style file for future PRs 👍